client/src/components/ImageUploader.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { BsPlusSquareDotted } from 'react-icons/bs'
+import styled from 'styled-components';
+
+const UploadContainer = styled.div`
+  font-size: 64px;
+  height: 400px;
+  width: 450px;
+  color: #999;
+  border: 3px solid #999;
+  border-radius: 10%;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+`;
+
+const ImageUploader = ({upload}) => {
+  const [selected, setSelected] = useState(false);
+
```

Comment:
Assuming the selection feature for this component that requires useState is just not fully implemented yet.

Reply:
Yep!